datacards/produce_datacard.sh

```diff
@@ -3,4 +3,11 @@
 source utils/setup_cmssw.sh
 source utils/setup_python.sh
 
-python datacards/produce_datacard.py --channels $@
+CHANNELS=$1
+VARIABLE=$2
```

Comment:
I think this doesn't work together with  run_analysis.sh if more than one channel is requested

Reply:
you're correct, we'll do it in two scripts